src/lib/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -0,0 +1,20 @@
+.QueryWrapperPlotNav {
+  .QueryWrapperPlotNav__title {
```

Comment:
you can just do 
.QueryWrapperPlotNav {
&__title {...}
&__querycount {...}
&__actions {}
}
 same thing functionally